megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -180,6 +179,35 @@ public CommonMenuBar() {
         fileUnitsPaste = createMenuItem(submenu, getString("CommonMenuBar.fileUnitsPaste"), FILE_UNITS_PASTE);
         fileUnitsPaste.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_V, InputEvent.CTRL_DOWN_MASK));
 
+        // Create the Board sub-menu.
+        menu = new JMenu(getString("CommonMenuBar.BoardMenu")); 
+        add(menu);
+        boardNew = createMenuItem(menu, getString("CommonMenuBar.fileBoardNew"), BOARD_NEW);
+        boardOpen = createMenuItem(menu, getString("CommonMenuBar.fileBoardOpen"), BOARD_OPEN);
+        boardSave = createMenuItem(menu, getString("CommonMenuBar.fileBoardSave"), BOARD_SAVE);
+        boardSaveAs = createMenuItem(menu, getString("CommonMenuBar.fileBoardSaveAs"), BOARD_SAVE_AS);
+        menu.addSeparator();
+        boardSaveAsImage = createMenuItem(menu, getString("CommonMenuBar.fileBoardSaveAsImage"), BOARD_SAVE_AS_IMAGE);
+        boardSaveAsImage.setToolTipText(getString("CommonMenuBar.fileBoardSaveAsImage.tooltip"));
+        boardSaveAsImageUnits = createMenuItem(menu, getString("CommonMenuBar.fileBoardSaveAsImageUnits"), BOARD_SAVE_AS_IMAGE_UNITS);
+        boardSaveAsImageUnits.setToolTipText(getString("CommonMenuBar.fileBoardSaveAsImageUnits.tooltip"));
+        menu.addSeparator();
+        boardUndo = createMenuItem(menu, getString("CommonMenuBar.boardUndo"), BOARD_UNDO); 
+        boardUndo.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_Z, InputEvent.CTRL_DOWN_MASK));
+        boardRedo = createMenuItem(menu, getString("CommonMenuBar.boardRedo"), BOARD_REDO);
```

Comment:
Can we do a control y for redo?

Reply:
I wanted to but there's a keycommandbind with Ctrl-Y for toggling the unit labels that eats that shortcut (I tried). We should maybe make a pass at the standard key settings at some point to make them more in line with computing after 1989. Then we can add the Ctrl-Y.